foosball/games/forms.py

```diff
@@ -38,3 +45,6 @@ class GameForm(SuperForm):
     game = ModelFormField(GameModelForm)
     team1 = ModelFormField(TeamModelForm)
     team2 = ModelFormField(TeamModelForm)
+
+    def is_valid(self):
+        return super().is_valid() & clean_team_forms(self.forms['team1'], self.forms['team2'])
```

Comment:
Actually it is intentionally `&` so that `clean_team_forms()` will be evaluated (and it's error messages added) also when `is_valid()` returns `False`. This way the functionality matches other validations; all error messages are shown at the same time. Maybe could have added a comment about that in the code.